lib/kennel/models/monitor.rb

```diff
@@ -76,8 +76,9 @@ def as_json
             escalation_message: Utils.presence(escalation_message.strip),
             evaluation_delay: evaluation_delay,
             locked: false, # setting this to true prevents any edit and breaks updates when using replace workflow
-            renotify_interval: renotify_interval || 0
-          }
+            renotify_interval: renotify_interval || 0,
+            renotify_statuses: notify_no_data ? ["alert", "no data"] : ["alert"]
```

Comment:
can we only set renotify_statuses if renotify_interval is not 0 ?
... I'm hoping that would produce less diff in the `rake generate` and get the same result
also it should be include warning when there is a warning configured ?

Reply:
sure ill change it to renotify_interval.

>  also it should be include warning when there is a warning configured ?
what does it mean?

Reply:
I think the alert state can be `warn`, `alert`, or `no data` so if they've got a warning alert threshold configured then renotify statuses might wanna be `["alert", "warn", "no data"]`

Reply:
ah i see. thanks ill add warn

Reply:
💇  @grosser